src/OpenTelemetry.AutoInstrumentation.Native/rejit_handler.cpp

```diff
@@ -119,15 +119,15 @@ void RejitHandlerModuleMethod::RequestRejitForInlinersInModule(ModuleID moduleId
         else if (hr == E_INVALIDARG)
         {
             Logger::Info("NGEN:: Error Invalid arguments in [ModuleId=", currentModuleId,
-                         ",MethodDef=", currentMethodDef, ", HR=", hr, "]");
+                         ",MethodDef=", currentMethodDef, ", HR=E_INVALIDARG]");
```

Comment:
Will it be serialized also as hex?

Reply:
This is a "string" and it will be serialized as `E_INVALIDARG` so it is even more readable  😉